lib/Common/Memory/RecyclerPointers.h

```diff
@@ -489,21 +489,26 @@ template <class Policy>
 struct _QuickSortImpl
 {
     template<class T, class Comparer>
-    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context)
+    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context, size_t elementSize = sizeof(T))
     {
+#ifndef _WIN32
```

Comment:
use GLOBAL_ENABLE_WRITE_BARRIER instead, and combine with ForceSoftwareWriteBarrier, so we can test this on windows?